README.md

```diff
@@ -47,7 +47,7 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 ### Launch
 
 - `mainClass` (required) - The main class of the program (fully qualified name, e.g. [mymodule/]com.xyz.MainClass).
-- `args` - The command line arguments passed to the program.
+- `args` - The command line arguments passed to the program. Support prompting for program arguments by using variable `"${command:SpecifyProgramArgs}"`.
```

Comment:
Use `"${command:SpecifyProgramArgs}"` to specify program arguments for each debug session.

Reply:
Feels like the new message doesn't reflect the prompting way to configure args. And not each debug session would take this way to configure args. This is an optional way. The `args` in launch.json still can be a hard-code constant string (previous design) or prompting.

How about `Use "${command:SpecifyProgramArgs}" to prompt user to specify program arguments dynamically.`?

Reply:
Use `"${command:SpecifyProgramArgs}"` to prompt for program arguments.